samples/SortBenchmark/DiskHelper.fs

```diff
@@ -291,3 +299,85 @@ type DiskHelper(records:int64) =
                 x.sorteddataSP.Save() 
                 b <- true
                 x.sorteddataSP.lock := 0
+
+
+
+[<AllowNullLiteral>]
```

Comment:
General comment: I am against "null" whenever possible. In F#, there's option. For public interface code, if really there's no way to construct objects that are "complete", we can use the null object pattern: https://en.wikipedia.org/wiki/Null_Object_pattern. Unfortunately, C# doesn't have a "[DisallowNullLiteral]" attribute, so ultimately one is always exposed. But generally avoiding null pointers at runtime will go a long way towards avoiding NullReferenceException. That's one of the most common kinds of exceptions, and it's a whole class of bugs that can largely mitigated.